src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeGeneration/CodeGenerationHelpers.cs

```diff
@@ -129,7 +129,7 @@ public static int GetPreferredIndex(int index, IList<bool>? availableIndices, bo
             var newLineStarter = string.Concat("\n", commentStarter);
 
             // Start the comment with an empty line for visual clarity.
-            comment = string.Concat(commentStarter, "\r\n", commentStarter, xml.Replace("\n", newLineStarter));
+            comment = string.Concat(commentStarter, "\r\n", commentStarter, xml!.Replace("\n", newLineStarter));
```

Comment:
We have a call to string.IsNullOrEmpty above.  But it appars as if nullability info isn't flowing along from that in this layer.  not sure why.

Reply:
CodeStyle layer is likely depending on older compiler bits, so probably a bug fixed recently in nullability analysis.